test_programs/execution_success/regression_10156/src/main.nr

```diff
@@ -1,21 +1,21 @@
 pub fn main() -> pub Field {
     println(foo); // normal
     println(bar); // oracle
-    println(std::as_witness); // intrinsic
-    println(std::hash::blake2s::<10>); // builtin
-
-    // safety:
-    unsafe {
-        unconstrained_take_bar(bar);
-    }
-    take_bar(bar);
-
-    // safety:
-    unsafe {
-        unconstrained_take_as_witness(std::as_witness);
-    }
-    take_as_witness(std::as_witness);
-    std::as_witness(foo());
+    // println(std::as_witness); // intrinsic
+    // println(std::hash::blake2s::<10>); // builtin
+
+    // // safety:
+    // unsafe {
+    //     unconstrained_take_bar(bar);
+    // }
+    // take_bar(bar);
```

Comment:
I understand the `println` had to be commented out for intrinsic and builtin, but can you remind me why these don't work?

Reply:
Right, thanks; I should have left some more comments to make it clear that I was trying to test passing intrinsics and built-ins as values, to oracles (print) and user functions as well.